unison-link.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -s /bin/unison-fsmonitor-$UNISON_VERSION /bin/unison-fsmonitor
```

Comment:
Let's move this into the `unison-install.sh` script, since this shouldn't need to be linked on every run.